app/controllers/miq_policy_controller.rb

```diff
@@ -284,8 +233,38 @@ def fetch_log
                        :message => "Policy log downloaded")
   end
 
+  # need to load toolbar here, because rsop screen code is in the same controller.
+  def show_list
+    super
+    @center_toolbar = "miq_policies"
```

Comment:
Don't we have a nicer way of declaring the toolbars? As far as I remember, this could be something like:
```
toolbar :miq_policy, :miq_policies
```
I'm sure there are examples in other controllers...

Reply:
@skateman this will not work in Policy controller because, Simulation, Log and Import/Export sub-meunu screens still live in this controller, which is why i need to define `@center_toolbar = "miq_policies"` appropriately

Reply:
Since these are resolved, can we use this `toolbar` now instead of hacking the toolbar choose over and over?

Reply:
@skateman this has already been resolved

Reply:
Yeah, but I don't see the `toolbar` DSL in the controller...

Reply:
https://github.com/ManageIQ/manageiq-ui-classic/pull/7401/files#diff-dbd39c7ca425617ef9b4bcd4e7e5e0be31135ba5c00b5ba13e6269e622e44f30R76